public/js/actions/subscriptions.js

```diff
@@ -33,43 +33,68 @@ export function getUserSubscriptions(jquery=$) {
 }
 
 
-export function createSubscription(braintreeToken, productId,
-                                   creditCard, jquery=$,
+export function createSubscription(productId, payMethod,
+                                   braintreeToken, jquery=$,
                                    BraintreeClient=braintree.api.Client) {
   return dispatch => {
 
-    var client = new BraintreeClient({
-      clientToken: braintreeToken,
-    });
+    function requestSub(opts={}) {
```

Comment:
nevermind, I see that either `payMethodUri` or `nonce` can be used. I still think params would be nice: `createSub({payMethodUri=undefined, nonce=undefined})`